.build/faqs/index.html

```diff
@@ -108,7 +108,7 @@ <h2 class="faqsBlock-heading">Overview</h2>
         <p><span class="leading">Q</span>What is the difference between coin mixing and CashShuffle?</p>
       </div>
       <div class="faqs-a">
-        <p><span class="leading">A</span>&quot;Coin mixing&quot; commonly refers to the use of services that allow a user to replace his or her coins with a different set of coins. CashShuffle is different. It allows users to combine their transactions with others, creating obfuscation.</p>
+        <p><span class="leading">A</span>"Coin mixing" commonly refers to the use of services that allow a user to replace his or her coins with a different set of coins. CashShuffle is different. It allows users to combine their transactions with others, creating obfuscation.</p>
```

Comment:
Hmm this is invalid HTML... these should be `&quot;` =\

Reply:
Since you are using `{{{` we need to make sure strings are HTML escaped correctly.

Reply:
Ok I fixed everything within my knowledge including the English and Japanese translations and the explanation in the translation documentation.

The switch to `{{{` raw strings in a lot of places may make other translations more difficult. I will keep an eye on it.